src/events.ts

```diff
@@ -59,12 +59,21 @@ export type DeviceDeletedEvent = CommonDeviceEvent<"device.deleted">
 
 export type DeviceThirdPartyIntegrationDetected =
   CommonDeviceEvent<"device.third_party_integration_detected">
+export type DeviceThirdPartyIntegrationNoLongerDetected =
+  CommonDeviceEvent<"device.third_party_integration_no_longer_detected">
```

Comment:
```suggestion
  CommonDeviceEvent<"device.third_party_integration_removed">
```

Reply:
CC @razor-x

Reply:
ehh i changed my mind on this one- the pairing is nice, and we don't want to imply that we are able to detect when a third party integration is always added